### PR TITLE
Adjust mobile breakpoints and fix different pages for mobile

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: pkg/storaged/fsys-panel.jsx [hunk 1]
```diff
@@ -169,6 +169,7 @@ export class FilesystemsPanel extends React.Component {
             <OptionalPanel id="mounts" className="storage-mounts"
                 title={_("Filesystems")}>
                 <ListingTable
+                    gridBreakPoint="grid-xl"
```
Comment (garrett) [inline pkg/storaged/fsys-panel.jsx:172]: Should this be a different breakpoint?

I still see unoptimal layouts like this:

image

It's definitely better with this PR (no overlaps), though!

Comment (garrett) [reply pkg/storaged/fsys-panel.jsx:172]: Oh, whoops. This isn't the right place (I've been playing with different values.) We'd want to change it elsewhere, I think.

**Review by garrett — APPROVED** [review]
I think this is so much better! Thanks!

I did spot a problem where we tried to squeeze the sidebar in when there just wasn't enough room. (I commented with a screenshot, but in the wrong area.)

I fixed this by removing the `md` sizes in a follow-up commit that I pushed to this PR.

WDYT?

**Review by martinpitt — APPROVED** [review]
Code-wise this LGTM, thanks!

**Pull request merged**